src/NuGet.Deterministic.Package/build/NuGet.Deterministic.targets

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask AssemblyFile="$(NuGetDeterministicTasksAssemblyPath)" TaskName="NuGet.Tasks.Deterministic" />
+
+  <PropertyGroup>
+    <PrepareForBuildDependsOn Condition=" '$(GenerateNuGetDeterministicProps)' == 'true' ">$(PrepareForBuildDependsOn);GenerateNuGetDeterministicProps</PrepareForBuildDependsOn>
+    <PrepareForBuildDependsOn Condition=" '$(ImportedNuGetDeterministicProps)' == 'true' ">$(PrepareForBuildDependsOn);ValidateNuGetDeterministic</PrepareForBuildDependsOn>
```

Comment:
Maybe rename `ImportedNuGetDeterministicProps` to `NuGetDeterministicPropsWasImported`